openehr-rm/src/main/java/com/nedap/archie/rminfo/UpdatedValueHandler.java

```diff
@@ -79,6 +81,12 @@ private static void fixDvOrdinal(Object rmObject, Archetype archetype, String pa
                 }
             }
         }
+
+        Map<String, Long> result = new HashMap<>();
+        String pathToValue = pathOfParent.replace("/symbol/defining_code", "/value");
```

Comment:
Why do you always add value, even if you do not set the value in the ordinal due to some of the if conditions?